src/confetti.js

```diff
@@ -286,8 +288,9 @@
     return {
       x: opts.x,
       y: opts.y,
-      wobble: Math.random() * 10,
-      wobbleSpeed: Math.min(0.11, Math.random() * 0.1 + 0.05),
+      disableWobble: opts.disableWobble,
+      wobble: opts.disableWobble ? 0 : Math.random() * 10,
+      wobbleSpeed: Math.random() * opts.wobbleSpeed,
```

Comment:
This seems to suggest that the value being passed in by the user will be multiplied by a random value, therefore just making the wobble speed a random value and not at all what the user passed in. This makes wobble speed unpredictable. That is not what is described in the README update. I've also only read the code, but I think this changes the default behavior from what was previously implemented as well. Did you intend for these two things to happen?